pyproject.toml

```diff
@@ -4,20 +4,20 @@ dynamic = ["version"]
 description = "The main tool to build, upload, and develop in general the Juju charms."
 readme = "README.md"
 dependencies = [
-    "craft-application~=3.0",
+    "craft-application>=4.0.0",
```

Comment:
This will also pick up 5.0, is it intended ?

Reply:
For now, yes. This is changed separately.